tests/lib/snaps/apply-config-hook-set/meta/hooks/apply-config

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "Setting a configuration value"
+if ! snapctl set baz=qux; then
```

Comment:
Oh, you were probably pointing out that _this_ snap doesn't do much at the moment. Indeed, but it'll grow in the next PR.